src/constants/ShoukakuConstants.js

```diff
@@ -109,6 +109,7 @@ class ShoukakuConstants {
     * @property {string} host Your node host / ip address of where the lavalink is hosted.
     * @property {number} port The Port Number of your lavalink instance.
     * @property {string} auth The authentication key you set on your lavalink config.
+    * @property {boolean} secure the protocols should secure or no.
```

Comment:
just wrap secure with [] and ? on start of boolean to signify its optional on docs